crates/starknet/src/starknet/dump.rs

```diff
@@ -129,9 +129,14 @@ impl Starknet {
         }
     }
 
-    /// save starknet transactions to file
     pub fn dump_transactions(&self) -> DevnetResult<()> {
-        match &self.config.dump_path {
+        self.dump_transactions_custom_path(None)
+    }
+
+    /// save starknet transactions to file
+    pub fn dump_transactions_custom_path(&self, custom_path: Option<String>) -> DevnetResult<()> {
+        let dump_path = if custom_path.is_some() { &custom_path } else { &self.config.dump_path };
```

Comment:
Can't `unwrap_or` be used here?

Reply:
I don't this so since dump_path is Option<String>